.github/workflows/push-ingestor-build.yaml

```diff
@@ -27,10 +27,16 @@ jobs:
           aws-region: ${{ secrets.AWS_REGION }}
           role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
           role-duration-seconds: 1200
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Build and push Docker image
-        env:
-          ECR_REPO: ${{ secrets.ECR_REPO }}
-          AWS_REGION: ${{ secrets.AWS_REGION }}
-        run: make push-ingestor-build ecr_repo=$ECR_REPO tag=latest aws_region=$AWS_REGION
+      - name: Docker build & Push
+        uses: docker/build-push-action@v4
+        with:
+          context: ./ingestion_tools
+          file: ./ingestion_tools/Dockerfile
+          tags: ${{ secrets.ECR_REPO }}:${{ github.head_ref || github.ref_name }}
```

Comment:
As this is triggered on push to main, `github.head_ref` might always turn up empty.

Reply:
I tested it on another branch, and the branch name came through, so 🤞 I'm expecting this to resolve to `main` in most cases